new-log-viewer/src/utils/actions.ts

```diff
@@ -67,42 +64,84 @@ const EDITOR_ACTIONS : ActionType[] = [
         keyBindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyI],
     },
 ];
-/* eslint-enable sort-keys */
```

Comment:
Intentional?

Reply:
Gotcha. It's part of a pair with the other half being on line 26. So if we remove one, we should remove the other.

I thought disabling the rule was intentional to avoid sorting the keys of each object, but it turns out the rule in our [config](https://github.com/y-scope/eslint-config-yscope/blob/main/common.js#L711) only enforces sorting if there >= 5 keys, so we don't need the disabling.

Reply:
Probably, but I think let's leave that to another PR or else this one will get too big.